tests/test_converters/test_obo_graph_to_fhir.py

```diff
@@ -206,16 +244,43 @@ def test_convert_so(self):
             )
             dl_path = OUTPUT_DIR / "so.owl.obographs.json"
             out = OUTPUT_DIR / f"{filename}.json"
-            cs: CodeSystem = self._load_and_convert(
+            cs: CodeSystem = self._load_and_convert_json(
                 out,
                 dl_path,
                 dl_url=dl_url,
                 code_system_id=filename.replace("CodeSystem-", ""),
                 code_system_url="http://purl.obolibrary.org/obo/so.owl",
                 native_uri_stems=["http://purl.obolibrary.org/obo/SO_"],
             )
-            # TODO: choose a better threshold
+            # todo: choose a better threshold
             self.assertGreater(len(cs.concept), 100)
-            # TODO: choose a property to assert
+            # todo: choose a property to assert
             # prop_uris: List[str] = [p.uri for p in cs.property]
             # self.assertIn("", prop_uris)
+
+    @unittest.skip("TODO: change to an integration test")
```

Comment:
@cmungall You added these `@unittest.skip("TODO: change to an integration test")` lines to most of the FHIR unit tests, but that erroneously makes it look like these are being skipped because and only because they should be integration tests instead. I think I'm going to change it to `@unittest.skip("Skipping test due to long run time due to downloads")  # TODO: change to an integration test`.